include/prism.h

```diff
@@ -147,6 +157,7 @@ void pm_serialize_content(pm_parser_t *parser, pm_node_t *node, pm_buffer_t *buf
  * @param parser The parser to serialize.
  * @param node The node to serialize.
  * @param buffer The buffer to serialize to.
+ * \public \memberof pm_node
```

Comment:
Why are we pointing to `pm_buffer_t` in the previous one and `pm_node` instead of `pm_node_t` in this one?